airbyte-integrations/connectors/source-harvest/source_harvest/source.py

```diff
@@ -54,11 +54,8 @@ def get_authenticator(config):
         credentials = config.get("credentials", {})
         if credentials and "client_id" in credentials:
             return HarvestOauth2Authenticator(
+                config,
                 token_refresh_endpoint="https://id.getharvest.com/api/v2/oauth2/token",
-                client_id=credentials.get("client_id"),
```

Comment:
why were these removed?

Reply:
The `client_id` parameter does not exist in my `SingleUseRefreshTokenOauth2Authenticator` implementation. The `client_id` is retrieved from the configuration with dpath